core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraXCameraUseCase.kt

```diff
@@ -560,12 +565,24 @@ constructor(
                 .start(callbackExecutor) { onVideoRecordEvent ->
                     run {
                         Log.d(TAG, onVideoRecordEvent.toString())
+                        val isTorchRequired = camera.cameraInfo.hasFlashUnit() &&
```

Comment:
What happens if we don't check if there is a flash unit? We like to remove this kind of check so that the app will crash if you try to enable the torch when it's not available. The UI should only allow developers to enable the torch if the camera constraints allow it.

Reply:
> What happens if we don't check if there is a flash unit?
The returned future will fail immediately since the camera control will also check the flash unit.

Change the code not to check the flash unit, so it will crash if there is no flash unit.